segments/tmux_session_info.sh

```diff
@@ -1,7 +1,24 @@
 # Prints tmux session info.
-# Assuems that [ -n "$TMUX"].
+# Assumes that [ -n "$TMUX"].
+
+TMUX_POWERLINE_SEG_TMUX_SESSION_INFO_FORMAT_DEFAULT='[ #S ]:#I.#P'
```

Comment:
I'd say let's keep the old format as default, to not change unnecessarily for people upgrading by git pull

Reply:
```suggestion
TMUX_POWERLINE_SEG_TMUX_SESSION_INFO_FORMAT_DEFAULT='#S:#I.#P'
```